### DISCUSSION TITLE
Security audit of zstd

### DISCUSSION THREAD

**Discussion by iio7** [openzfs/zfs#11615] [General]
Has anyone done any kind of security audit, outside of Facebook, on zstd?

Considering all the privacy problems with Facebook, and possibly associations with three letter agencies, I would like to know how serious this code has been reviewed before it where made default compression algorithm in ZFS?

**Comment by Derkades** [comment]
What are your concerns? It certainly does not secretly make HTTP requests sending your data to facebook or whatever, that would be trivial to spot.
